core/eolearn/tests/test_core_tasks.py

```diff
@@ -343,24 +343,46 @@ def test_zip_features_fails(patch: EOPatch) -> None:
 
 
 @pytest.mark.parametrize(
-    "feat_spc_org, feat_spc_map, fun",
+    "feature_spcification_original, feature_spcification_mapped, map_function",
```

Comment:
you have a lot of typos in various names, run a spellchecker

Reply:
run a spellchecker